app/contracts/queries/payment_receipts_query_filters_contract.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Queries
+  class PaymentReceiptsQueryFiltersContract < Dry::Validation::Contract
+    UUID_REGEX = /\A[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\z/
```

Comment:
This REGEX is already defined in `PaymentsQueryFiltersContract`. I think it should be extracted somewhere. I'm not sure where to be honest.
Maybe a new `app/support/regex.rb` and we can move `EmailValidator::EMAIL_REGEXP` there too.